# System Load Message
std_msgs/Header header
float32 cpu_percent
float32[] cpu_percent_per_core
float32 memory_percent
float32 memory_used_gb
float32 memory_total_gb
float32 swap_percent
float32[3] load_average  # 1min, 5min, 15min
